qemu/tests/qemu_img.py

```diff
@@ -509,40 +516,75 @@ def rebase_test(cmd):
                                                ignore_status=True):
             raise error.TestNAError("Current kvm user space version does not"
                                     " support 'rebase' subcommand")
+        base_fmt = params.get("base_format", "qcow2")
         sn_fmt = params.get("snapshot_format", "qcow2")
-        sn1 = params["image_name_snapshot1"]
-        sn1 = _get_image_filename(sn1, enable_gluster, sn_fmt)
-        base_img = storage.get_image_filename(params, data_dir.get_data_dir())
-        _create(cmd, sn1, sn_fmt, base_img=base_img, base_img_fmt=image_format)
-
-        # Create snapshot2 based on snapshot1
-        sn2 = params["image_name_snapshot2"]
-        sn2 = _get_image_filename(sn2, enable_gluster, sn_fmt)
-        _create(cmd, sn2, sn_fmt, base_img=sn1, base_img_fmt=sn_fmt)
+        skip_boot = params.get("skip_boot", "no")
+        lv_name_list = params.get("lv_name_list", "")
+        vg_name = params.get("vg_name", "")
+        backing_file_snapshot1 = params.get("backing_file_snapshot1", "base_img")
+        backing_file_snapshot2 = params.get("backing_file_snapshot2", "sn1")
+        sn1 = params.get("image_name_snapshot1", "images/sn1")
+        sn2 = params.get("image_name_snapshot2", "images/sn2")
+        if lv_name_list:
+            lv_size = params.get("lv_size", "")
```

Comment:
It's better to add steps info here , such as "setup lvmdevice: lv_base,lv_sn1,lv_sn2" ;